cloud/docker/docker_image.py

```diff
@@ -35,6 +35,12 @@
     required: false
     default: null
     aliases: []
+  dockerfile:
+    description:
```

Comment:
no, since 1.9 was already released, normally its the 'next version to release', 2.0 in this case